platypus/parse.py

```diff
@@ -263,20 +270,19 @@ def process_results(percentage_ids, alignment_lengths, percentage_ids_other,
             # Comparing bit_scores to create outputs
             if vals['a']['bit_score'] == vals['b']['bit_score']:
                 results[i]['equal'] += 1
-                results[i]['summary'].append('%s\t%s\t%s' % (seq_name,
-                                                             subject_id_a,
-                                                             subject_id_b))
+                results[i]['summary_fh'].write('%s\t%s\t%s\n' % (
+                    seq_name, subject_id_a, subject_id_b))
                 db_seqs_counts_a[subject_id_a] += 1
                 db_seqs_counts_b[subject_id_b] += 1
             elif vals['a']['bit_score'] > vals['b']['bit_score']:
                 if not subject_id_b:
                     results[i]['perfect_interest'] += 1
-                    results[i]['summary'].append('%s\t%s\t' % (seq_name,
-                                                               subject_id_a))
+                    results[i]['summary_fh'].write('%s\t%s\t\n' % (
```

Comment:
the header on line 239 has 3 columns but this is only writing two. Would it be possible to write an explicit null value so the resulting file is not jagged?